guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -0,0 +1,121 @@
+[id="configuring-the-active-directory-authentication-source-on-projectserver_{context}"]
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Prerequisites
```

Comment:
The prerequisites section is unusually long. I don't like that but I still think it's better than turning it into a separate procedure module. It's still a single prerequisite ("the system must be enrolled into the AD domain under a certain configuration") and adding the two commands is designed to make sure the instructions are clear enough.

Reply:
I wish I didn't have to specify the list of packages and the command to join AD explicitly like this (referring to RHEL docs would be the preferred choice because the RHEL IdM team is in sync with the engineering team developing these use cases) but unfortunately, Foreman requires a specific setup that combines SSSD and Samba. So the list of packages and the command to join the domain is different from what the RHEL docs recommend.

Reply:
I don't like commands in prerequisites. In most cases, it's possible to move the commands to the beginning of the procedure or to a separate procedure module (and then link it).